app/forms/registration.rb

```diff
@@ -99,7 +98,14 @@ def attendee_number
   private
 
   def attendee_params(params)
-    atrs = ATD_ATRS.concat(["birth_date(1i)", "birth_date(2i)", "birth_date(3i)", "user_id"])
+    if not admin?
+      # Remove admin-only permitted attributes
+      atrs = ATD_ATRS - [:comment, :minor_agreement_received]
```

Comment:
We subtract any admin-only attributes from the params during submission if the user isn't an admin.